providers/aws/resources/aws_cloudtrail.go

```diff
@@ -240,11 +240,31 @@ func (a *mqlAwsCloudtrailTrail) eventSelectors() ([]interface{}, error) {
 
 	arnValue := a.Arn.Data
 	// no pagination required
-	trailmgmtevents, err := svc.GetEventSelectors(ctx, &cloudtrail.GetEventSelectorsInput{
+	eventSelectorsOutput, err := svc.GetEventSelectors(ctx, &cloudtrail.GetEventSelectorsInput{
 		TrailName: &arnValue,
 	})
 	if err != nil {
 		return nil, err
 	}
-	return convert.JsonToDictSlice(trailmgmtevents.EventSelectors)
+
+	// Basic event selectors
```

Comment:
i think the issue is actually that we're not finding the trail you have this on. ill dig into it tonight